media-libs/panda3d/panda3d-1.9.2.ebuild

```diff
@@ -0,0 +1,98 @@
+# Copyright 2008-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python{2_7,3_4,3_5} )
+
+inherit eutils multiprocessing python-single-r1
```

Comment:
inheriting eutils was inherited (sorry, sometimes i cant help myself) from ezod. removed.
multiprocessing is needed as I need makeopts_jobs from it to pass to makepanda's --threads